app/controllers/poems_controller.rb

```diff
@@ -16,9 +16,8 @@ def new
   end
 
   def create
-    @poem = current_user.poems.build(poem_params)
-    if @poem.save
-      notify_to_slack
+    @poem = PoemCreationService.create(current_user, poem_params, view_context: view_context)
```

Comment:
__[rubocop]__
* Line is too long. [93/80] [:ref](https://github.com/bbatsov/ruby-style-guide#80-character-limits)